pgcrypto/fields.py

```diff
@@ -23,9 +23,17 @@ class TextDigestField(HashMixin, models.TextField):
     """Text digest field for postgres."""
     encrypt_sql = DIGEST_SQL
 
-    def get_encrypt_sql(self):
+    @staticmethod
+    def get_key(connection):
+        """Get key from connection or default to settings."""
```

Comment:
Since `get_setting` is also needed in `HashMixin` I went to simple helper method that doesn't live in any class. Resolved. Thanks for the refactor suggestion